src/test/java/com/sap/sgs/phosphor/fosstars/model/value/DateValueTest.java

```diff
@@ -53,6 +53,7 @@ public void serialization() throws IOException {
     ObjectMapper mapper = new ObjectMapper();
     DateValue dateValue = new DateValue(new DateFeature("test"), new Date());
     byte[] bytes = mapper.writeValueAsBytes(dateValue);
+    System.out.println(new String(bytes));
```

Comment:
do you still need it? I guess you added this for yourself right? :)

Reply:
Good catch! I was going to remove it but I forgot.